app/src/main/java/vn/edu/usth/minigh/DiscussionActivity.java

```diff
@@ -60,14 +53,26 @@ private CommentFragment generateComment() {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        TextView txt_toolbar = (TextView) findViewById(R.id.main_text_bar);
-        txt_toolbar.setText("Create profile activity");
+
+        String title = getIntent().getStringExtra("title");
+        TextView ghname = (TextView) findViewById(R.id.discussTitle);
+        ghname.setText(title);
+
+        String descrip = getIntent().getStringExtra("description");
+        TextView content = (TextView) findViewById(R.id.discussDescription);
+        content.setText(descrip);
+
         int nComments = 7;
         for (int i = 0; i < nComments; i++) {
             Fragment fragment = generateComment();
             getSupportFragmentManager().beginTransaction().setReorderingAllowed(true).add(
                     R.id.comment_container, fragment
             ).commit();
         }
+
+    }
+
+    public void backClick(View view) {
+        super.onBackPressed();
```

Comment:
I'm not convinced that we need a custom back button when it's part of the Android DE.

Reply:
I use this to turn back instead of open the drawer when click the button at the top left

Reply:
The DE's back button should work I think, given you give the custom the exact same handler.

Reply:
The DE's back button works. But if the `backClick` is not necessary, I'll remove them

Reply:
I'm more concern with the necessity of the button, which takes the place of the big mac.